docs/workforce-management-framework/topics/pro-using-demo-app.adoc

```diff
@@ -0,0 +1,110 @@
+The demo application can define business process or procedure via a custome Workflow and assign these workflows as jobs to
+individual workers as workorders. Workers can view their workorder via a mobile applicaion and administrators
+can managed workorders via the web portal.
+
+== Mobile application
+
+The mobile application is used by a end user to interact with workorders that have been assigned to them.
+login screen requires username and password. For a list of users see 
+link:https://github.com/feedhenry-raincatcher/raincatcher-core/blob/master/demo/server/src/modules/wfm-user/users.json[users.json],
+all users in the demo have a default password is `123`
+
+image::/shared/images/login.png[Login]
+
+Once logged in you will be presented with the workorders screen which will show the workorders assigned to the logged in user.
+In this example workorders are accident reports to be filled out at the seen of an accident.
+
+image::/shared/images/Workorder-mobile.png[Workorders]
+
+To log out click on the menu icon in the top left hand side of the screen this will open a sidebar with options to Logout or return
+to the Workorders screen.
+
+image::/shared/images/sidebar.png[Workorders]
+
+Return to the Workorders screen and click on an one of the workorders. The following screen will render
+
+image::/shared/images/workorder-overview.png[overview]
+
+Click on `BEGIN WORKFLOW` to proceed with data entry in the following `Accident Report`.
+
+image::/shared/images/workorder-accidentReport.png[accident report]
+
+Once the `Accident Report` form is completed click on `CONTINUE` on the bottom of the screen to proceed to the 
+`Vehicle Inspection` form. The form asks for checks for fuel ,tires and lights.
+
+image::/shared/images/workorder-vehicleInspection.png[vehicle inspection]
+
+Once the `Vehicle Inspection` form is completed click on `CONTINUE` on the bottom of the screen to proceed to the 
+`Summary` which shows the status of the workorder.
+
+image::/shared/images/workorder-summary.png[summary]
+
+Once on the summary screen you have finished your submission . To proceed click on the menu icon in the top left hand side of the screen this will open a sidebar with options to Logout or return
+to the Workorders screen. On the Workorders screen you will see that the workorder is now marked with a green tick to so completion.
+
+== Portal application
+
+The Portal appliction is used by a administrator to view , edit and assign Workorder and to create new workflows.
+The Portal application is a web application. The view of workorders is restricted to administrators. 
+The login screen is as follows
+
+image::/shared/images/login-portal.png[login portal]
+
+Onced logged in you will be presented with the following screen for admistration of workorders and workflows.
+
+image::/shared/images/portal.png[portal]
+
+From this screen you can create new workorders by clicking the `NEW WORKORDER` button on the right panal. Once clicked
```

Comment:
Present ideas gradually in the way people will need to setup them.
Workflows are the first! Presenting workorders before workflows makes no sense.